### PR TITLE
Add support for snake_case option in the pagination

### PR DESCRIPTION
**Pull request by brunoocasali**
Fix https://github.com/meilisearch/meilisearch-rails/issues/183

After this PR: https://github.com/meilisearch/meilisearch-rails/pull/49/files, we started to support snake_case options only in some attributes related to the Meilisearch engine like `sortableAttributes`.
And here https://github.com/meilisearch/meilisearch-rails/pull/154, I updated all the non-snake case attributes massively without checking if there was a missing spot like this.

TL;DR: 

`hitsPerPage` and `hits_per_page` should work fine now.

### CODE REVIEW

**Review by mdubus — APPROVED** [review]
LGTM 🚀
